src/thunderbots/software/ai/navigator/obstacle/velocity_obstacle.h

```diff
@@ -0,0 +1,27 @@
+#pragma once
+
+#include "ai/navigator/obstacle/obstacle.h"
+#include "shared/constants.h"
+
+// Placeholder obstacle represents a stationary robot centred at (0,0)
```

Comment:
Placeholder? I think we should replace this with a detailed description of what this class is. Right now I'm still pretty confused.